tests/dummy/app/components/time-block.js

```diff
@@ -8,11 +8,23 @@ export default Ember.Component.extend({
 
 	humanizedDifference: Ember.computed('difference', function () {
 		if (this.get('difference') === 0) {
-			return this.get('i18n').t('global.now');
+			if (this.get('i18n').exists('global.now')) {
+				return this.get('i18n').t('global.now');
+			} else {
+				return 'now';
+			}
 		} else if (this.get('difference') < 0) {
-			return this.get('i18n').t('global.ago', {seconds: -this.get('difference')});
+			if (this.get('i18n').exists('global.ago')) {
+				return this.get('i18n').t('global.ago', {seconds: -this.get('difference')});
+			} else {
+				return `ago this.get('difference')`;
```

Comment:
These test are looking exactly on the not-exists branch, that's why it checked `ago -10` and not `AGO_10`.